src/Resources/skeleton/command/Command.tpl.php

```diff
@@ -10,6 +10,11 @@
 )]
 class <?= $class_name; ?> extends Command
 {
+    public function __construct()
```

Comment:
Should we have this?

```suggestion
    public function __construct(string $name = null)
```